### ISSUE TITLE
False positive PSR1.Files.SideEffects.FoundWithSymbols when using php8 attribute

### ISSUE THREAD

**Issue by dakorpar** [squizlabs/PHP_CodeSniffer#3283]
**Describe the bug**
Simple attribute class gives warning

**Code sample**
```php
<?php declare(strict_types = 1);

namespace My\Namespace;

#[\Attribute]
class Cached
{

}

```

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="My Custom Standard">
  <description>If you are using a custom ruleset, please enter it here.</description>
 <rule ref="PSR1.Files.SideEffects.FoundWithSymbols"></rule
</ruleset>
```

**To reproduce**
3. See error message displayed
```
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 1 | WARNING | A file should declare new symbols (classes, functions,
   |         | constants, etc.) and cause no other side effects, or
   |         | it should execute logic with side effects, but should
   |         | not do both. The first symbol is defined on line 6 and
   |         | the first side effect is on line 5.
```

**Expected behavior**

**Versions (please complete the following information):**
 - OS: [e.g., Windows 10, MacOS 10.15]
 - PHP: [8.0]
 - PHPCS: [dev-master]
 - Standard: [PSR1]

**Comment by jrfnl** [comment]
As per #3182 and #3203, attributes are not yet supported in PHPCS, not even in `dev-master`. Testing of #3203 could help move it forward, but without that PR (or an iteration of it) being merged, it won't be possible to start making sniffs compatible with attributes yet.

**Comment by gsherwood** [comment]
Now that attribute support has been merged, I've updated this sniff to ignore attributes, which fixes this problem. Thanks for reporting it.
